test/test_helper.rb

```diff
@@ -200,8 +200,8 @@ def assert_valid(model)
     def refute_with_errors(condition, model, field = nil, match = nil)
       refute condition, "#{model.inspect} errors: #{model.errors.full_messages.join(';')}"
       if field
-        assert_blank model.errors.map { |a,m| model.errors.full_message(a, m) unless field == a }.compact
-        assert_present model.errors[field].find { |e| e.match(match) },
+        assert model.errors.map { |a,m| model.errors.full_message(a, m) unless field == a }.compact.blank?
```

Comment:
Can you add a custom message here, with the contents of this map.compact call?  I actually relied on this in a failing test earlier today, and the output from a plain `assert` won't be anywhere near as helpful.  At least assert_present says something like "actual X does not match expected Y", while assert'll just say "assertion failed".